tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Send Email.3540.feature

```diff
@@ -21,23 +21,20 @@ Scenario: 1. User receives email after their account is created
 		| FirstName | LastName   | PhoneNumber | EmailAddress             | OrganisationName |
 		| Bob       | Bobkovitch | 0123456789  | bob.bobkovitch@email.com | Organisation 1   |
 	Then a response with status code 200 is returned
-	And an email containing the following information is sent
-		| From                 | To                       | Subject          | Body                                                                                                      |
-		| admin.user@email.com | bob.bobkovitch@email.com | Account Creation | Hello, please click this link to complete your registration http://nhsd.create.users.co.uk/b0bsS3cr3tL1nk |
-	And the sent email contains the following information
-		| From                 | To                       | Subject          | Body                                          |
-		| admin.user@email.com | bob.bobkovitch@email.com | Account Creation | http://nhsd.create.users.co.uk/b0bsS3cr3tL1nk |
+	And sent email contains the following information
+		| From                           | To                       | Subject                           | ResetPasswordLink         |
+		| noreply@buyingcatalogue.nhs.uk | bob.bobkovitch@email.com | Set password for Buying Catalogue | https://www.google.co.uk/ |
 
 @3540
 Scenario: 2. User does not receive email when non-authority user tries to create their account
 	Given an user is logged in
```

Comment:
I've noticed this one before and was tempted to change that. Then I realised that it would mean a lot of extra changes to this PR and decided against it.
I'm happy to change it in this change set as long as you don't mind reviewing the extra files :]

Reply:
That's fine. I don't mind given that they won't be functional changes.